test/numbers.jl

```diff
@@ -830,6 +830,47 @@ end
     @test cmp(isless, 1, NaN) == -1
     @test cmp(isless, NaN, NaN) == 0
 end
+@testset "ispositive/isnegative" begin
+    @test ispositive(1)        == true
+    @test ispositive(0)        == false
+    @test ispositive(-1)       == false
+    @test ispositive(1.0)      == true
+    @test ispositive(0.0)      == false
+    @test ispositive(-0.0)     == false
+    @test ispositive(-1.0)     == false
+    @test ispositive(1.0/0.0)  == true
```

Comment:
This is redundant with `ispositive(Inf)` below

Reply:
I copied the tests from
https://github.com/putianyi889/julia/blob/b4266e897d0a74d828906b4b2b7f4999786a8b66/test/numbers.jl#L538-L556
same holds for some of your comments below.

Do you think it's a good idea to use a different style than the rest of the file?

Reply:
Well, yes, I don't see how `f(1.0/0.0)` is testing anything different than `f(Inf)`.

Reply:
Would you like to delete those from other tests?